### PR TITLE
fix for isort issue #104 and yamllint

### PR DESCRIPTION
**Pull request by worroc**

### CODE REVIEW

File: captainhook/checkers/yamllint_checker.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# # # # # # # # # # # # # #
+# CAPTAINHOOK IDENTIFIER  #
+# # # # # # # # # # # # # #
+import os
+from .utils import bash, get_config_file
+
+DEFAULT = 'off'
+CHECK_NAME = 'yamllint'
+NO_YAMLLINT_MSG = ("yamllint is required for the yamllint plugin.\n"
+                "`pip install yamllint` or turn it off in your {} file.".format(get_config_file()))
```
Comment (OddBloke) [inline captainhook/checkers/yamllint_checker.py:10]: This line looks mis-indented (and long).

File: captainhook/checkers/yamllint_checker.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+REQUIRED_FILES = ['.yamllint']
+
+
+def _filter_yaml_files(files):
```
Comment (OddBloke) [inline captainhook/checkers/yamllint_checker.py:14]: The comments indicate that this was copy/pasted; should we instead have this in `.utils` so both the Python, YAML and future checkers can use the same code path?

**Review by OddBloke — CHANGES_REQUESTED** [review]

**Comment by OddBloke** [comment]
(If you could split the isort change to a separate PR, that's easier to accept and would be much appreciated. :smile: )

**Comment by worroc** [comment]
I will do all the staff in separate pull request

**Pull request closed without merging**